source/usr/lib/mcvirt/cluster/cluster.py

```diff
@@ -197,15 +234,76 @@ def checkRemoteMachine(self, remote_object):
                 raise RemoteObjectConflict('Remote node contains duplicate Virtual Machine: %s' %
                                            local_vm)
 
+        # If DRBD is enabled on the local machine, ensure it is installed on the remote machine
+        # and is not already enabled
+        from mcvirt.node.drbd import (DRBD as NodeDRBD,
+                                      DRBDNotInstalledException,
+                                      DRBDAlreadyEnabled)
+        if (NodeDRBD.isEnabled()):
+            if (not remote_object.runRemoteCommand('node-drbd-isInstalled', [])):
+                raise DRBDNotInstalledException('DRBD is not installed on the remote node')
+
+            if (remote_object.runRemoteCommand('node-drbd-isEnabled', [])):
+                raise DRBDNotInstalledException('DRBD is already enabled on the remote node')
+
     def removeNode(self, remote_host):
         """Removes a node from the MCVirt cluster"""
         # Ensure the user has privileges to manage the cluster
         self.mcvirt_instance.getAuthObject().assertPermission(Auth.PERMISSIONS.MANAGE_CLUSTER)
-        remote = self.getRemoteNode(remote_host)
-        remote.runRemoteCommand('cluster-cluster-removeNodeConfiguration',
-                                {'node': self.getHostname()})
+
+        # Ensure node exists
+        self.ensureNodeExists(remote_host)
+
+        # Check for any VMs that the node to be removed is available to and is not the only
```

Comment:
Is the following better?

    # Check for any VMs that the target node is available to and where the node is not the only